### DISCUSSION TITLE
Aggiornamento registro (PROPOSTA)

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#330] [Idee e proposte]
Buonasera ,ho pensato tanto a tutti i colleghi/tecnici che non riescono ogni volta ad aggiornare il registro all'uscita di una nuova versione o alla sistemazione di un qualsiasi problema  ,nonostante le guide e non i video (che , a parer mio farebbero capire più da vicino il tutto a tutti ).A questo punto ho pensato che , sarebbe bello ,come avete implementato la funzione automatica del passaggio all'anno successivo ,implementare nel registro stesso (lato amministratore) una funzione denominata (aggiorna alla versione più recente) in modo che cliccando lì ,si aggiorni in automatico ,senza ogni volta dover fare tutto dal database .In questo modo si risparmierebbe tempo a tutti

**Comment by scuolagioco** [comment]
Buonasera, non sarebbe male come idea anche perché sono uno che non riesce ad aggiornare ogni volta il registro😂😂, però non credo sia fattibile.. però lasciamo la conferma a @trinko

**Comment by trinko** [comment]
Ci sto pensando da un po'... E' fattibile, ma richiede molto tempo per lo sviluppo, perché le cose che possono andare storte sono tante.
Comunque, appena trovo un po' di tempo, ci provo.

**Comment by sempliciniluigi89** [comment]
Che bello sentire tanta positività e tanto spirito di iniziativa,mi rasserena

**Comment by ISConvertini2022** [comment]
Salve,
C'è per caso qualche utente disposto ad aiutarci a caricare semplicemente la cartella di installazione sul nostro Server ed a fare in modo che sia possibile utilizzare il Registro, al resto penseremo ovviamente noi, ma questo è un passaggio che stranamente sta creando problemi, veniamo da LampSchool e di questi problemi nell'installazione non ne abbiamo avuti.
Grazie infinite,
Team Digital
IIS 'N. Convertini'

**Reply by manuelbarberi** [reply]
Si, mi scriva alla mail giuaschoolregistro@gmail.com
